user/pages/04.Reference/03.compute/docs.en.md

```diff
@@ -268,5 +268,5 @@ The security group "default" is added in this example, since this group is takin
 Any occupied resources are considered "used" and counted against your [quota limits](../../05.Background/03.resource-limits/docs.en.md#usage). They will also be counted against your upfront commitment or charged on demand.
 When you create an instance, volume, snapshot, object, loadbalancer, floating ip, dns zone, resources will be allocated and so they will be counted and charged, no matter if they are actually running, attached, assigned.
 To save money or free quota, it is thus not sufficient to stop, detach or unassign them, you must delete them completely to free the resources so that we can reuse them.
-It is part of higher level automation like kubernetes or terraform or similar tools to recreate them if needed.
+It is part of higher level automation like Kubernetes or terraform or similar tools to recreate them if needed.
```

Comment:
Either `kubernetes or terraform` or `Kubernetes or Terraform` but not mixed, please.